app/assets/stylesheets/components/_native-campaign.scss

```diff
@@ -0,0 +1,37 @@
+$light-grey: #bfc1c3;
+$dark-grey: #999;
+
+.native-campaign {
+  display: block;
+  border-top: 1px solid $light-grey;
+  padding: 20px 0 0;
+}
+
+.native-campaign span {
```

Comment:
We should use BEM naming conventions instead of applying styles to elements.
See this example: https://github.com/alphagov/government-frontend/blob/master/app/assets/stylesheets/components/_contents-list.scss#L29